### ISSUE TITLE
the benefit of hinge loss

### ISSUE THREAD

**Issue by IPNUISTlegal** [taki0112/Self-Attention-GAN-Tensorflow#10]
![image](https://user-images.githubusercontent.com/35527568/43630992-776b501c-9734-11e8-89a0-87e415b4f04b.png)

i find citation 13,16,30 and do not know exact principle of hinge loss.
i feel confused about why don't we use WGAN loss function.
cause it has better performance than WGAN loss function?
thx
